Sources/Subs.php

```diff
@@ -543,16 +543,22 @@ function updateSettings($changeArray, $update = false)
  * @param int $num_per_page The number of items to be displayed on a given page. $start will be forced to be a multiple of this value.
  * @param bool $flexible_start Whether a ;start=x component should be introduced into the URL automatically (see above)
  * @param bool $show_prevnext Whether the Previous and Next links should be shown (should be on only when navigating the list)
+ * @param null|array $options Provide max_id and low_id to evade paging by next page jump
  *
  * @return string The complete HTML of the page index that was requested, formatted by the template.
  */
-function constructPageIndex($base_url, &$start, $max_value, $num_per_page, $flexible_start = false, $show_prevnext = true)
+function constructPageIndex($base_url, &$start, $max_value, $num_per_page, $flexible_start = false, $show_prevnext = true, $options = null)
 {
 	global $modSettings, $context, $smcFunc, $settings, $txt;
 
 	// Save whether $start was less than 0 or not.
 	$start = (int) $start;
 	$start_invalid = $start < 0;
+	
+	if (!empty($options['low_id']))
+		$low_id = $options['low_id'];
```

Comment:
Reason why i said to me that i don't need this,
because i say in query side that is had to be a int: https://github.com/SimpleMachines/SMF2.1/pull/4134/files/e2b10fd24660a764ae3f5566800eb29b49348231#diff-eeb7dcd590ea5ad96bd502e7c294017bR889
but maybe not and bad idea to force here too